src/content/docs/tutorial-customer-experience/establish-current-state.mdx

```diff
@@ -0,0 +1,168 @@
+---
+title: "Establish your current state" 
+metaDescription: "New Relic observability maturity series: this guide gives you a quality foundation for optimizing your web application's performance to improve your customer experience."
+redirects:
+---
+
+import cxSegmentWhitelistInvestigation from 'images/cx_screenshot-full_segment-whitelist-investigation.webp'
+
+import cxQfDashboard from 'images/cx_screenshot-full_qf-dashboard.webp'
+
+import cxQfKpiProgress from 'images/cx_diagram_qf_kpi_progress.webp'
+
+The first step in any improvement experience is to establish your baselines so that you have numbers to measure your KPIs against and can track improvement over time, and improving your customer experience is no exception to this rule. Once you check to make sure everything you need is reporting correctly to our platform, you can validate your information and use our quality foundation dashboard to measure your customer experience.
+
+<Steps>
+    <Step>
+### Review instrumented pages
+
+First, you'll need to review your browser apps and pages to make sure that everything you expect to report to New Relic is actually doing so. You can do this by reviewing the **Page Views** tab in the browser monitoring UI or running the following NRQL query:
+
+```
+SELECT uniques(pageUrl) from PageView LIMIT MAX 
+```
+
+<Callout variant="tip">
+New to NRQL? Check out our [introduction to NRQL](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language/) to learn about how to use our custom query language to help maximize your data!
+</Callout>
+
+    </Step>
+    <Step>
+### Validate browser URL grouping
+
+After you've made sure that your instrumented pages report their data correctly, it's time to ensure browser segments are captured correctly. If you're unfamiliar with segments, they're simply the text between two `/`s in a URL or the `.`s of a domain name. For example, in the URL `website.com/product/widget-name`, the segments are:
+* `website`
```

Comment:
I'm not totally sure we need this bulleted list of examples. I think the explanation above it plenty clear.

Reply:
No problem, always happy to save real estate! Thank you!